crates/matrix-sdk-common/src/linked_chunk/builder.rs

```diff
@@ -45,33 +234,30 @@ struct TemporaryChunk<Item, Gap> {
 /// then the final call to [`LinkedChunkBuilder::build()`] will result in an
 /// error).
 #[allow(missing_debug_implementations)]
-pub struct LinkedChunkBuilder<const CAP: usize, Item, Gap> {
+#[doc(hidden)]
+pub struct LinkedChunkBuilderTest<const CAP: usize, Item, Gap> {
```

Comment:
If it's only used in tests, consider removing it? :fire:

Reply:
As discussed, will be done as an immediate follow up.